.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+*       @cloudevents/maintainers
```

Comment:
I don't think we want all CE SDK maintainers to have write access, do we?

Reply:
yup, that's what I wanted to discuss. will change 👍

Reply:
I mean... do we really trust @deissnerk ? :-)